crates/rethnet_evm_napi/src/receipt.rs

```diff
@@ -0,0 +1,12 @@
+use napi::bindgen_prelude::{BigInt, Buffer};
+use napi_derive::napi;
+
+use crate::Log;
+
+#[napi(object)]
+pub struct Receipt {
```

Comment:
This doesn't seem to be used, am I missing something?

Reply:
You're 100% right. We don't actually output the receipt from Rethnet (yet). I can remove it if you'd like?

Reply:
No, it's fine if it's going to be used :)